src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import books from './books/books';
+import categories from './categories/categories';
+
+const reducer = combineReducers({ books, categories });
+
+const store = configureStore(
+  reducer,
+);
+
+export default store;
```

Comment:
- [OPTIONAL] 🤔 What If you Import `applyMiddleware`  to use some handy Redux middleware, like a redux logger.  it will help you a lot in the development process. 👍🏻 To install it, you need to run: 

  ```  npm i --save redux-logger  ```